### PR TITLE
PrincipalContext.ValidateCredentials method and username argument

### PR DESCRIPTION
**Pull request by rpetrusha**
# PrincipalContext.ValidateCredentials method and username argument

## Summary

The documentation incorrectly states that the `username` argument can be in any of a number of formats. It cannot.

## Suggested Reviewers

//cc @BillWagner @Cloud-Writer

### CODE REVIEW

**Comment by rpetrusha** [comment]
@BillWagner, @mairaw, @JRAlexander, please don't merge this for me once it's approved. I'll take care of merging it, since it requires PSS review.

**Review by BillWagner — APPROVED** [review]
LGTM.  Let’s :shipit:

**Pull request merged**
